src/map/clif.c

```diff
@@ -11098,6 +11098,10 @@ static void clif_parse_LoadEndAck(int fd, struct map_session_data *sd)
 		if (battle_config.show_tip_window != 0)
 			clif->open_ui_send(sd, ZC_TIPBOX_UI);
 #endif
+#if PACKETVER_RE_NUM >= 20180307 || PACKETVER_MAIN_NUM >= 20180404 || PACKETVER_ZERO_NUM >= 20180411
+		if ((battle_config.feature_enable_attendance_system != 0) && (clif->attendance_timediff(sd)))
```

Comment:
```suggestion
		if (battle_config.feature_enable_attendance_system != 0 && clif->attendance_timediff(sd) == true)
```
btw is it possible to make it optional to popup on every login if the reward aren't redeemed?
Maybe the user would like to redeem it at a later time and already acknowledged the time is completed?

Reply:
@Emistry Hmm it just works like the tipbox UI and will just load upon character login, when it popup and you didn't redeem, it will not popup again unless your switch character or relogin the said character.

Reply:
tipsbox are configurable...
https://github.com/HerculesWS/Hercules/commit/ddf679b3808e56309a7352eead2cd1f93c9f99c9#diff-ff11d33dedaf6be65dc557a335aa2d55

Reply:
@Emistry 
that is configured by the server owner, that could be added, will commit it in a while
Maybe the user would like to redeem it at a later time and already acknowledged the time is completed? - I'm referring to this one as I thought you want it to be a player option